spec/unit/lib/cloud_controller/dependency_locator_spec.rb

```diff
@@ -484,6 +484,41 @@
 
       expect(locator.statsd_client).to eq(expected_client)
     end
+
+    it 'returns the null statsd client' do
+      host = 'test-host'
+      port = 1234
+
+      TestConfig.override(
+        statsd_host: host,
```

Comment:
For this test case host and port are not needed. Removing them makes the test shorter and easier to read.